client/client-multi/client-objective-c/Kaa/persistance/KaaClientPropertiesState.m

```diff
@@ -81,13 +83,18 @@ @implementation KaaClientPropertiesState
 @synthesize needProfileResync = _needProfileResync;
 
 - (instancetype)initWithBase64:(id<KAABase64>)base64 clientProperties:(KaaClientProperties *)properties {
+    return [self initWithBase64:base64 clientProperties:properties autogeneratedKeys:NO];
```

Comment:
I'd make the default strategy to generate keys for the user but not expect them to be already generated.